ZenPacks/zenoss/Import4/model.py

```diff
@@ -143,6 +143,12 @@ def zenpack(self):
 
         log.info("zenpacks restored:%s", codeString[ExitCode.SUCCESS])
 
+        # copy the license files for ucspm if exist
```

Comment:
I would prefer that we put this with the database move.  It's more tightly coupled to that than to zenpacks.